cloud/kyup/kyup.py

```diff
@@ -0,0 +1,357 @@
+      - Kyup.com encryption key. The key can also be supplied by setting the Environment variable KYUP_ENC_KEY.
+    required: false
+  action:
+    description:
+      - Action you want to perform on the container.
+    required: true
+  name:
+    description:
+      - Name of the container that you will create/operate on
+    required: true
+  ssh_keys:
+    description:
+      - Comma separated list of ssh key names, that have access to this container.
+    required: false
+  password:
+    description:
+      - Password for the root user of the container. Required when I(action) = 'create'.
+    required: false
+  image:
+    description:
+      - Template image to use when creating the container. Required when I(action) = 'create'.
+    required: false
+  dc_id:
+    description:
+      - DataCenter location ID. Required when I(action) = 'create'.
+    required: false
+    default: 1
+  mem:
+    description:
+      - Memory of the container in GB. Minimum 1 GB.
+    required: false
+    default: 1
+  hdd:
+    description:
+      - Storage size in GB. Minimum 20 GB.
+    required: false
+    default: 20
+  cpu_cores:
+    description:
+      - Number of CPU cores to be used. Minimum 2 CPU cores.
+    required: false
+    default: 2
+  bw:
+    description:
+      - Bandwidth limit for this container, in TB. Minimum 2 TB.
+    required: false
+    default: 2
+  storage_type:
+    description:
+      - Choose the storage type for this container.
+    required: false
+    default: local
+'''
+
+EXAMPLES = '''
+# Basic provisioning example
+---
+- hosts: kyup
+  tasks:
+  - name: ensure kyup container
+    kyup: >
+      api_key=XXXX
+      enc_key=YYYY
+      action=create
+      image="Centos Plain"
+      dc_id=1
+      name=avalon
+      password=e4a17265b28f6a9e8657
+      ssh_keys=hackman,piele
+  - add_host: name={{ container_ip }} group=containers
+
+- hosts: containers
+  tasks:
+  - name: copy the hosts file
+    copy: src=/etc/hosts dest=/etc/hosts
+
+
+# Rebooting a container
+---
+- hosts: kyup
+  tasks:
+  - name: restart the container
+    kyup: >
+      api_key=XXXX
+      action=restart
+      name=avalon
+
+'''
+
+RETURN = '''
+kyup_container_name:
+    returned: success
+    type: string
+    sample: "container_test"
+kyup_container_ip:
+    returned: success
+    type: string
+    sample: "10.12.131.5"
+'''
+
+import os
+import time
+import base64
+import md5
+from Crypto.Cipher import AES
+
+# import module snippets
+from ansible.module_utils.basic import *
+from ansible.module_utils.urls import *
+
+## Defaults for the module
+# API URL
+url='https://api.kyup.com/client/v1'
+
+# Modify this counter if you want to wait for the requests, more then a minute
+retries=12 # 12 x 5sec = 60 seconds
+
+
+## Code used for encryption of the passwords
+blocksize = AES.block_size
+
+def pad( data ):
+    padding = blocksize - ( len(data) % blocksize )
+    return data + (chr(padding) * padding)
+
+def encrypt( key, data ):
```

Comment:
Can you please remove the whitespace around these params to fit with PEP8 code style?